services/blkmgr/handle_block.go

```diff
@@ -145,7 +145,7 @@ func (b *BlockManager) handleBlockMsg(bmsg *blockMsg) {
 		*/
 		isCurrent := b.IsCurrent()
 		if isCurrent {
-			log.Info(fmt.Sprintf("Your synchronization has been completed. "))
+			log.Debug("Your synchronization has been completed. ")
```

Comment:
The information level changes may not be easy to accept from the user experience since the completion is 
 important information for them.

Reply:
OK